backend/src/actions/grafana/createPromQLQueries.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @description export functions that create promql queries with a passed in container ID
```

Comment:
NIT: we're not using JSDocs anywhere else in our project so IMO we can remove this comment. I think the filename you've given this is descriptive enough!